transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystem10Parser.java

```diff
@@ -198,11 +198,8 @@ static void parseCoreEnvironmentElement(final XMLExtendedStreamReader reader, fi
                     TransactionSubsystemRootResourceDefinition.NODE_IDENTIFIER.parseAndSetParameter(value, operation, reader);
                     break;
                 case PATH:
-                    TransactionSubsystemRootResourceDefinition.PATH.parseAndSetParameter(value, operation, reader);
-                    break;
                 case RELATIVE_TO:
-                    TransactionSubsystemRootResourceDefinition.RELATIVE_TO.parseAndSetParameter(value, operation, reader);
-                    break;
+                    throw new XMLStreamException("the attribute '" + attribute.getLocalName() + "' is no longer supported", reader.getLocation());
```

Comment:
This needs to be done in an i18n manner, with a new method in the org.jboss.as.txn.logging.TransactionLogger class.

Also, please make it "The attribute".